### ISSUE TITLE
Center loss values grows exponentially and becomes nan after infinity

### ISSUE THREAD

**Issue by srikarreddy** [davidsandberg/facenet#980]
Following are the training logs 
Epoch: [1][73/1000]	Time 32.095	Loss 7.075	Xent 0.001	RegLoss 7.074	Accuracy 1.000	Lr 0.05000	**Cl 570070614378393063228374893199360.000**
Epoch: [1][74/1000]	Time 32.244	Loss 7.075	Xent 0.001	RegLoss 7.074	Accuracy 1.000	Lr 0.05000	**Cl** **1814146193983481504114967901634560.000**
Epoch: [1][75/1000]	Time 32.216	Loss 7.077	Xent 0.001	RegLoss 7.076	Accuracy 1.000	Lr 0.05000	**Cl 5218579563508895537146879635619840.000**
Epoch: [1][76/1000]	Time 32.242	Loss 7.074	Xent 0.001	RegLoss 7.073	Accuracy 1.000	Lr 0.05000	**Cl inf**
.
.
.

/home/srikar/.local/lib/python2.7/site-packages/numpy/lib/histograms.py:824: **RuntimeWarning: invalid value encountered in greater_equal**
  keep = (tmp_a >= first_edge)
/home/srikar/.local/lib/python2.7/site-packages/numpy/lib/histograms.py:825: **RuntimeWarning: invalid value encountered in less_equal**
  keep &= (tmp_a <= last_edge)
Epoch: [1][164/1000]	Time 32.122	Loss nan	Xent nan	RegLoss nan	Accuracy 0.578	Lr 0.05000	**Cl nan**

**Comment by fisakhan** [comment]
I guess the size of the training data is very small. Isn't it?

**Comment by koxt** [comment]
https://github.com/davidsandberg/facenet/issues/820#issuecomment-479373419 
the same issue

**Comment by jjsjunior** [comment]
I had this issue. In my case the problem was not shuffling training set.
